pkgs/development/python-modules/wandb/default.nix

```diff
@@ -137,6 +138,18 @@ let
     versionCheckProgramArg = "--version";
     doInstallCheck = true;
 
+    checkFlags =
+      let
+        skippedTests = [
+          # gpu sampling crashes in the sandbox
+          "TestSystemMonitor_BasicStateTransitions"
+          "TestSystemMonitor_RepeatedCalls"
+          "TestSystemMonitor_UnexpectedTransitions"
+          "TestSystemMonitor_FullCycle"
+        ];
+      in
+      [ "-skip=^${lib.concatStringsSep "$|^" skippedTests}$" ];
```

Comment:
Does `buildGoModule` not have an attribute analogous to `buildPythonPackage`'s `disabledTests`?

Reply:
Last time I asked (~2024-05) that was the recommended pattern.

Reply:
Gotcha, works for me